modules/inspecting-pod-and-container-logs.adoc

```diff
@@ -42,7 +42,25 @@ Logs retrieved using the preceding `oc logs` commands are composed of messages s
 +
 [source,terminal]
 ----
-$ oc exec <pod_name> ls -alh /var/log
+$ oc exec <pod_name>  -- ls -alh /var/log
+----
++
+.Example output
+[source,terminal]
```

Comment:
Per the latest [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) to avoid unhelpful highlighting:
```suggestion
[source,text]
```